.kokoro/release/common.cfg

```diff
@@ -9,7 +9,7 @@ build_file: "gapic-generator-java/.kokoro/trampoline.sh"
 # Configure the docker image for kokoro-trampoline.
 env_vars: {
   key: "TRAMPOLINE_IMAGE"
-  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
```

Comment:
gapic-generator-java will still require release with java8, I know we are already overriding this to java8 in gapic-generator-java submodule, just want to take a note here so that we don't forget this.